test/BaselineOutput/SingleDebug/MulticlassLogisticRegression/LogisticRegression-Non-Negative-CV-iris-out.txt

```diff
@@ -53,50 +53,56 @@ Virtual memory usage(MB): %Number%
-[2] (%Time%)	6 iterations	Loss: 0.281388521194458	Improvement: 0.06186
-[2] (%Time%)	7 iterations	Loss: 0.237996473908424	Improvement: 0.04801
-[2] (%Time%)	8 iterations	Loss: 0.212298363447189	Improvement: 0.03128
-[2] (%Time%)	9 iterations	Loss: 0.199792444705963	Improvement: 0.0172
-[2] (%Time%)	10 iterations	Loss: 0.194789424538612	Improvement: 0.008052
-[2] (%Time%)	11 iterations	Loss: 0.193230450153351	Improvement: 0.003182
-[2] (%Time%)	12 iterations	Loss: 0.192447692155838	Improvement: 0.001383
-[2] (%Time%)	13 iterations	Loss: 0.189304739236832	Improvement: 0.002703
-[2] (%Time%)	14 iterations	Loss: 0.187662661075592	Improvement: 0.001907
-[2] (%Time%)	15 iterations	Loss: 0.185374572873116	Improvement: 0.002193
-[2] (%Time%)	16 iterations	Loss: 0.18364554643631	Improvement: 0.001845
-[2] (%Time%)	17 iterations	Loss: 0.180794909596443	Improvement: 0.002599
-[2] (%Time%)	18 iterations	Loss: 0.178908497095108	Improvement: 0.002065
-[2] (%Time%)	19 iterations	Loss: 0.175620675086975	Improvement: 0.002982
-[2] (%Time%)	20 iterations	Loss: 0.174758642911911	Improvement: 0.001392
-[2] (%Time%)	21 iterations	Loss: 0.173962101340294	Improvement: 0.0009454
-[2] 'LBFGS Optimizer' finished in %Time%.
-[3] 'LBFGS data prep #2' started.
-[3] 'LBFGS data prep #2' finished in %Time%.
-[4] 'LBFGS Optimizer #2' started.
-[4] (%Time%)	0 iterations	Loss: 1.0986123085022
-[4] (%Time%)	1 iterations	Loss: 1.05856335163116	Improvement: 0.04005
-[4] (%Time%)	2 iterations	Loss: 1.00281620025635	Improvement: 0.05261
-[4] (%Time%)	3 iterations	Loss: 0.97780430316925	Improvement: 0.03158
-[4] (%Time%)	4 iterations	Loss: 0.752716302871704	Improvement: 0.1773
-[4] (%Time%)	5 iterations	Loss: 0.542387366294861	Improvement: 0.2021
-[4] (%Time%)	6 iterations	Loss: 0.443084180355072	Improvement: 0.125
-[4] (%Time%)	7 iterations	Loss: 0.343867212533951	Improvement: 0.1057
-[4] (%Time%)	8 iterations	Loss: 0.284590691328049	Improvement: 0.07087
-[4] (%Time%)	9 iterations	Loss: 0.254261910915375	Improvement: 0.04046
-[4] (%Time%)	10 iterations	Loss: 0.224356189370155	Improvement: 0.03255
-[4] (%Time%)	11 iterations	Loss: 0.215291574597359	Improvement: 0.01493
-[4] (%Time%)	12 iterations	Loss: 0.212821274995804	Improvement: 0.005586
-[4] (%Time%)	13 iterations	Loss: 0.212086588144302	Improvement: 0.001948
-[4] (%Time%)	14 iterations	Loss: 0.21061946451664	Improvement: 0.001587
-[4] (%Time%)	15 iterations	Loss: 0.209799557924271	Improvement: 0.001012
-[4] (%Time%)	16 iterations	Loss: 0.209267094731331	Improvement: 0.0006523
-[4] 'LBFGS Optimizer #2' finished in %Time%.
+[1] 'Building term dictionary' started.
+[1] (%Time%)	71 examples	Total Terms: 3
+[1] 'Building term dictionary' finished in %Time%.
+[2] 'LBFGS data prep' started.
+[2] 'LBFGS data prep' finished in %Time%.
+[3] 'LBFGS Optimizer' started.
+[3] (%Time%)	0 iterations	Loss: 1.0986123085022
+[3] (%Time%)	1 iterations	Loss: 1.00646448135376	Improvement: 0.09215
```

Comment:
>1.00646448135376	Improvement: 0.09215 [](start = 32, length = 37)

it starts differing here..

Reply:
different seed I would assume because of added transform.
It converge to same metrics at least.

---
In reply to: [263166427](https://github.com/dotnet/machinelearning/pull/2804#discussion_r263166427) [](ancestors = 263166427)

Reply:
actually it is fine. 

---
In reply to: [263168167](https://github.com/dotnet/machinelearning/pull/2804#discussion_r263168167) [](ancestors = 263168167,263166427)